jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -172,8 +173,15 @@ protected void setFilterSettings(final FilterSettings filter, final Consumer<Lis
     }
 
     public ActiveFilterItem getActiveFilterFromColumnFilter(ColumnFilter columnFilter) {
+        String lableValue = columnFilter.toString();
+        if (columnFilter instanceof CoreFunctionFilter) {
+            lableValue = ((CoreFunctionFilter) columnFilter).getLabelValue();
+            if (lableValue == null || lableValue.length() == 0) {
```

Comment:
lableValue.length()==0 -> labelValue.isEmpty()

Reply:
JDK's code in here 
    public boolean isEmpty() {
        return value.length == 0;
    }

Reply:
And is there any particular reason why you're not sticking with isEmpty() method provided by JDK, and you're using length() method for comparison? Either way there's method invocation - either length() or isEmpty(), you're not saving any overhead (cause not working directly with the length field). IMO the isEmpty() method usage improves readability

Reply:
I also refer to apach common's code https://github.com/apache/commons-lang/blob/master/src/main/java/org/apache/commons/lang3/StringUtils.java#L3528. 
But, we can not use commons in client code.

Reply:
updated

Reply:
Empty means null and length is zero, so if arrange Empty and null, it's little confusing.

Reply:
In what scent? You have a String variable x ... you want to check if it is an empty string .... so first you do null check, only then invoke method. .... which part is confusing?

Reply:
I mean, empty include null check and length is zero.

Reply:
Are we talking about the same thing?
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#isEmpty()
You need to null-check before invoking a method on an object.

Reply:
Usually, we use StringUtils of apache-commons for check String. You can see
the [code](https://github.com/kiegroup/jbpm-wb/blob/a34552c62202b8c6741cb170173b27ae8b08528e/jbpm-wb-forms/jbpm-wb-forms-backend/src/main/java/org/jbpm/workbench/forms/display/backend/provider/KieWorkbenchFormsValuesProcessor.java#L135). So, for me, empty include null check and length check.